go/tasks/pluginmachinery/k8s/client.go

```diff
@@ -0,0 +1,133 @@
+package k8s
```

Comment:
Lets add a comment to the top of the file, that will automatically get generated as a godoc :). I know it did not exist, but been trying to do it everytime i touch a file